index.js

```diff
@@ -57,10 +57,10 @@ function getElAll(selector) {
 const bookmarks = getElAll('.card__bookmark')
 //console.log(bookmarks)
 
-//Answerbuttons
+// Answerbuttons
 const answerButtons = getElAll('.card__button')
 
-//Toogle Answer
+// Toogle Answer
 for (let i = 0; i < answerButtons.length; i++) {
```

Comment:
How about using `forEach` here just to practice modern array methods? I think it would be much cleaner with `forEach` than a `for` loop.